src/components/Inputs/WrappedColorPicker.tsx

```diff
@@ -42,6 +42,22 @@ const StyledColorPickerTrigger = styled(Button)<{ $theme: AppTheme; $open: boole
   }
 `;
 
+const ColorPickerContainer = styled.div`
+  & .ant-color-picker .ant-color-picker-alpha-input {
+    & > .ant-input-number-input-wrap > .ant-input-number-input.ant-input-number-input {
```

Comment:
repeated for specificity?

Reply:
yup, unfortunately. I'll see if I can make that a little less egregious